### ISSUE TITLE
Names with parenthesis are not matched

### ISSUE THREAD

**Issue by tuirgin** [vtt-lair/token-replacer#97]
Filename matches fail if the actor name and image have parenthesis.

I have the following image in the directory configured as token-replacer's root:

- Giant_Beetle_(4HD).svg

In addition I added a few others when troubleshooting:

- Giant_Beetle,_4HD.svg
- Giant_Beetle_[4HD].svg
- Giant_Beetle_{4HD}.svg

The only match that fails is `Giant_Beetle_(4HD).svg`. I added a log statement to show the contents of the cache, and you can see that of the 4 filenames above, the comma and brackets are all escaped, while the parens are unaltered:

```
token-replacer.js:891 Token Replacer: Caching root folder: 'data', 'modules/tuirgins-tokens/tokens/dd-glyphs'
token-replacer.js:922 Token Replacer: tr_cachedTokens:
modules/tuirgins-tokens/tokens/dd-glyphs/Giant_Beetle%2C_4HD.svg,modules/tuirgins-tokens/tokens/dd-glyphs/Giant_Beetle_(4HD).svg,modules/tuirgins-tokens/tokens/dd-glyphs/Giant_Beetle_%5B4HD%5D.svg,modules/tuirgins-tokens/tokens/dd-glyphs/Giant_Beetle_%7B4HD%7D.svg
```

However, when the attempt to match occurs, token-replacer is looking for a pattern with escaped parens:

```
token-replacer.js:709 Token Replacer: searching for token for giant_beetle_%284hd%29
token-replacer.js:710 Token Replacer: searching for portrait for portrait_giant_beetle_%284hd%29
```

Here's an export of my console log from which the above code blocks were taken: [202204281019-token_replacer-parenthesis.log](https://github.com/vtt-lair/token-replacer/files/8587732/202204281019-token_replacer-parenthesis.log)

**Comment by vtt-lair** [comment]
Thanks @tuirgin, I'll have a look at this in the following week. Sorry I've been so quiet, I immigrated and the last 5 months has been hectic.

**Comment by vtt-lair** [comment]
This should have been fixed with 0.7.1. Note that it's v10 only though
